db/tag.h

```diff
@@ -115,7 +115,8 @@ typedef enum convert_errcode {
     CONVERT_FAILED_INVALID_PARTIAL_TYPE,
     CONVERT_FAILED_BAD_BLOB_PROGRAMMER,
     CONVERT_FAILED_INPUT_TAG_HAS_INVALID_FIELD,
-    CONVERT_FAILED_INDEX_EXPRESSION
+    CONVERT_FAILED_INDEX_EXPRESSION,
+    CONVERT_FAILED_BLOB_SIZE
```

Comment:
(Optional) I prefer ending the last entry with a comma, so that next addition will result in a single-line diff.